lib/Echidna/Campaign.hs

```diff
@@ -219,13 +219,18 @@ execTxOptC :: (MonadState x m, Has Campaign x, Has VM x, MonadThrow m) => Tx ->
 execTxOptC t = do
   og  <- hasLens . coverage <<.= mempty
   res <- execTxWith vmExcept (usingCoverage $ pointCoverage (hasLens . coverage)) t
-  hasLens . coverage %= unionWith union og
+  let vmr = isRevert $ fst res
```

Comment:
we should have view patterns for reversions

Reply:
I believe this will be generalized soon with a different value for every type of revert.

Reply:
Done.